roles/core/pxe_stack/templates/nodes_parameters.yml.j2

```
@@ -19,15 +30,14 @@
   {% if hostvars[host]['equipment_profile']['equipment_type'] == "server" %}
     {% if hostvars[host]['network_interfaces'] is defined and hostvars[host]['network_interfaces'] is not none %}
 {{ host }}:
-  equipment_profile: {{ hostvars[host]['group_names'] | select('match','^'+equipment_naming+'_.*') | list | unique | sort | first | join | replace(equipment_naming + '_','') | trim }}
+  equipment_profile: {{ hostvars[host]['group_names'] | select('match','^'+equipment_naming+'_.*') | list | unique | sort | first | replace(equipment_naming + '_','') | trim }}
   network:
-{% set host_node_main_network = (macro_network.node_main_network(hostvars[host]['network_interfaces'],j2_current_iceberg_network)| trim) %}
-{% set host_node_main_network_interface = (macro_network.node_main_network_interface(hostvars[host]['network_interfaces'],host_node_main_network) | trim) %}
-    node_main_network_interface: {{ host_node_main_network_interface }}
+{% set host_node_main_network = node_main_network(hostvars[host]['network_interfaces'],j2_current_iceberg_network) | trim %}
+    node_main_network_interface: {{ node_main_network_interface(hostvars[host]['network_interfaces'],j2_current_iceberg_network) | trim }}
```

Following the macro change above:
```suggestion
    node_main_network_interface: {{ host_node_main_network_interface['interface'] | trim }}
```